cmd/launcher/options.go

```diff
@@ -240,6 +255,7 @@ func shortUsage() {
 	printOpt("osqueryd_path")
 	fmt.Fprintf(os.Stderr, "\n")
 	printOpt("autoupdate")
+	printOpt("control")
```

Comment:
missed a newline

Reply:
it's intentional, i wanted to put the kill switches together